.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: python
+python:
+  - "2.7"
```

Comment:
Tohle generuje PIO? Myslel jsem, že dávno zmigrovali na Python3

Reply:
To je z Jarkova PR, kde vycházel z nějakého PIO templatu.

Reply:
@JarekParal

Reply:
Ano, tohle je template přímo vygenerovaný PIO (mám za to). Pokud si dobře pamatuji, tak PIO přecházelo na Python 3 někdy na přelomu roku 2019/2020, Podle [aktuální dokumentace](https://docs.platformio.org/en/latest/core/installation.html#system-requirements) již Python 2 nedoporučují. Takže asi bude lepší to přepsat na Python 3, což by měla být jen změna na odkazovaném řádku [na Python 3.5+](https://docs.platformio.org/en/latest/core/installation.html#prerequisites).

Reply:
@JarekParal Ok můžeš to udělat pls?

Reply:
Opraveno.

PS: Víte že v PR lze dělat "code suggestion", který pak lze přímo z PR commitnout. Docela pěkná feature:
![image](https://user-images.githubusercontent.com/6726556/81070095-15a3d400-8ee3-11ea-94cc-d746f3accabc.png)

Reply:
Cool